r/src/simulation_step.r

```diff
@@ -275,7 +277,8 @@ simulation_step <- function(before_footprint = list(function() {output}),
       }
       
       # Find necessary met files
-      met_files <- find_met_files(r_run_time, met_file_format, n_hours, met_path)
+      met_files <- find_met_files(r_run_time, n_hours, n_hours_per_met_file, 
+                                  met_file_format, met_path)
```

Comment:
Would this second call to `find_met_files` be better placed in the previous `if (met_subgrid_enable) {` conditional block?

Reply:
Agreed. Will do.